lib/internal/Magento/Framework/HTTP/Adapter/Curl.php

```diff
@@ -176,8 +176,12 @@ public function write($method, $url, $http_ver = '1.1', $headers = [], $body = '
         } elseif ($method == \Zend_Http_Client::GET) {
             curl_setopt($this->_getResource(), CURLOPT_HTTPGET, true);
             curl_setopt($this->_getResource(), CURLOPT_CUSTOMREQUEST, 'GET');
+        } elseif ($method == \Zend_Http_Client::DELETE) {
```

Comment:
Is there any reason why we can't use something like this?
```suggestion
        } elseif ($method === \Zend_Http_Client::DELETE) {
```

Reply:
No any reason, I have just followed same method check as it is there already for other method type.

Reply:
I just added in all places strict comparison